### PR TITLE
Layout

### PR DESCRIPTION
**Pull request by Gani4ka**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1,6 @@
+DB_HOST=https://nodejs-lists-app.vercel.app/
+
+NEXT_PUBLIC_API_URL=https://nodejs-lists-app.vercel.app/
+
+NEXT_PUBLIC_TOKEN=
```
Comment (natalya-sv) [inline .env]: Does this work? One of my projects I had to use the token without quotes. Just NEXT_PUBLIC_TOKEN=token_example

**Review by natalya-sv — APPROVED** [review]

**Review by natalya-sv — APPROVED** [review]
Cool! 👯

File: .env (hunk 1 above)
Comment (Gani4ka) [reply .env]: it wasn't work without quotes for me)) It is interesting. I will recheck and tell you.

Comment (Gani4ka) [reply .env]: Yeap, it works only with quotes. Other way I get an error with 'jwt must be provided' message

**Pull request merged**